src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +47,14 @@ export interface IPatientViewPageProps {
     clinicalDataStatus?: RequestStatus;
 }
 
+export function updatePatientPageTitle(ids: { patientId?:string, sampleId?:string }){
```

Comment:
Im sort of bike-shedding here maybe but...
I feel like it would be more idiomatic/cleaner to have `@computed get pageTitle` and just have a reaction that sets it on change